external/iotivity/iotivity_1.2-rel/resource/csdk/connectivity/src/tcp_adapter/catcpserver.c

```diff
@@ -745,6 +745,7 @@ static CAResult_t CATCPCreateSocket(int family, CATCPSessionInfo_t *svritem)
     if (connect(fd, (struct sockaddr *)&sa, socklen) < 0)
     {
         OIC_LOG_V(ERROR, TAG, "failed to connect socket, %s", strerror(errno));
+		close(fd);
```

Comment:
looks good. 
code indent.
And if 'fd' needs close at line 760?

Reply:
I don't know the role of CAWakeUpForReadFdsUpdate(). but I think it's dangerous close the socket when it fails to write unlike connect()